content/en/docs/v3.5/downgrades/downgrade_3_5.md

```diff
@@ -0,0 +1,276 @@
+
+#### Rollback
+
+If any member has been downgraded to v3.4, the cluster will be downgraded to v3.4, and the cluster and its operations will be "v3.4". You would need to follow the [Upgrade etcd from 3.4 to 3.5](../../upgrades/upgrade_3_5/) instructions to rollback.
+
+Please [download the snapshot backup](../../op-guide/maintenance/#snapshot-backup) to make downgrading the cluster possible even after it has been completely downgraded.
+
+### Downgrade procedure
+
+This example shows how to downgrade a 3-member v3.5 etcd cluster running on a local machine.
+
+#### Step 1: check downgrade requirements
+
+Is the cluster healthy and running v3.5.x?
+
+```bash
+etcdctl --endpoints=localhost:2379,localhost:22379,localhost:32379 endpoint health
+<<COMMENT
+localhost:2379 is healthy: successfully committed proposal: took = 2.118638ms
+localhost:22379 is healthy: successfully committed proposal: took = 3.631388ms
+localhost:32379 is healthy: successfully committed proposal: took = 2.157051ms
+COMMENT
+
+curl http://localhost:2379/version
+<<COMMENT
+{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}
+COMMENT
+
+curl http://localhost:22379/version
+<<COMMENT
+{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}
+COMMENT
+
+curl http://localhost:32379/version
+<<COMMENT
+{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}
+COMMENT
+```
+
+#### Step 2: download snapshot backup from leader
+
+[Download the snapshot backup](../../op-guide/maintenance/#snapshot-backup) to provide a downgrade path should any problems occur.
+
+etcd leader is guaranteed to have the latest application data, thus fetch snapshot from leader:
+
+```bash
+curl -sL http://localhost:2379/metrics | grep etcd_server_is_leader
+<<COMMENT
+# HELP etcd_server_is_leader Whether or not this member is a leader. 1 if is, 0 otherwise.
+# TYPE etcd_server_is_leader gauge
+etcd_server_is_leader 1
+COMMENT
+
+curl -sL http://localhost:22379/metrics | grep etcd_server_is_leader
+<<COMMENT
+etcd_server_is_leader 0
+COMMENT
+
+curl -sL http://localhost:32379/metrics | grep etcd_server_is_leader
+<<COMMENT
+etcd_server_is_leader 0
+COMMENT
+
+etcdctl --endpoints=localhost:2379 snapshot save backup.db
+<<COMMENT
+{"level":"info","ts":1526585787.148433,"caller":"snapshot/v3_snapshot.go:109","msg":"created temporary db file","path":"backup.db.part"}
+{"level":"info","ts":1526585787.1485257,"caller":"snapshot/v3_snapshot.go:120","msg":"fetching snapshot","endpoint":"localhost:2379"}
+{"level":"info","ts":1526585787.1519694,"caller":"snapshot/v3_snapshot.go:133","msg":"fetched snapshot","endpoint":"localhost:2379","took":0.003502721}
+{"level":"info","ts":1526585787.1520295,"caller":"snapshot/v3_snapshot.go:142","msg":"saved","path":"backup.db"}
+Snapshot saved at backup.db
+COMMENT
+```
+
+#### Step 3: stop one existing etcd server
```

Comment:
Here I would recommend avoiding stopping the leader to prevent 1-2 second downtime. This can be done by using move-leader command.

Reply:
done